server/src/main/kotlin/org/jetbrains/bsp/bazel/server/sync/ProjectResolver.kt

```diff
@@ -68,7 +68,7 @@ class ProjectResolver(
     val rootTargets = buildAspectResult.bepOutput.rootTargets().let { formatTargetsIfNeeded(it) }
     val targets = measured(
       "Parsing aspect outputs"
-    ) { targetInfoReader.readTargetMapFromAspectOutputs(aspectOutputs) }
+    ) { targetInfoReader.readTargetMapFromAspectOutputs(aspectOutputs).let { it.mapKeys { it.key.replace("@@", "@") } } }
```

Comment:
Here we instead normalize anything to have single @, which should be safe since this seems to be only a target lookup structure.